lego/apps/files/tests/test_api.py

```diff
@@ -44,3 +44,128 @@ def test_post_create_file_call(self, mock_create_file):
         except Exception:
             pass
         mock_create_file.assert_called_with(key, self.user, False)
+
+
+class SetSaveForUserTest(BaseAPITestCase):
+    fixtures = ["test_users.yaml", "test_files.yaml", "test_abakus_groups.yaml"]
+    key = "abakus.png"
+    token = "token"
+    url = f"/api/v1/files/{key}/set_save_for_use/"
+
+    def setUp(self):
+        self.webkom_user = User.objects.get(pk=9)
+        self.random_user = User.objects.get(pk=1)
```

Comment:
What does `random_user` mean? Is it a normal user? If so preferable use a name that reflects the object :)